pro/src/components/pages/Offerers/Offerer/VenueV1/fields/PricingPointV2/PricingPointV2.tsx

```diff
@@ -0,0 +1,173 @@
+import Button from 'ui-kit/Button/Button'
+import { Select } from 'ui-kit/form'
+import { Banner } from 'ui-kit/index'
+
+import styles from './PricingPointV2.module.scss'
+
+export interface IPricingPointProps {
+  readOnly: boolean
+  offerer: GetOffererResponseModel
+  venue: GetVenueResponseModel
+  setVenueHasPricingPoint: (venueHasPricingPoint: boolean) => void
+}
+
+const PricingPointV2 = ({
+  readOnly,
+  offerer,
+  venue,
+  setVenueHasPricingPoint,
+}: IPricingPointProps) => {
+  const [canSubmit, setCanSubmit] = useState(true)
+  const [isInputDisabled, setIsInputDisabled] = useState(false)
+  const [isConfirmSiretDialogOpen, setIsConfirmSiretDialogOpen] =
+    useState(false)
+  const [isBannerVisible, setIsBannerVisible] = useState(true)
+  const [pricingPointSelectField] = useField({ name: 'venueSiret' })
+
+  useEffect(() => {
+    setCanSubmit(!pricingPointSelectField.value)
+  }, [pricingPointSelectField.value])
+
+  const handleClick = async () => {
+    const pricingPointId = pricingPointSelectField.value
+    if (venue?.id) {
+      api
+        .linkVenueToPricingPoint(venue.id, {
+          pricingPointId: pricingPointId,
+        })
+        .then(() => {
+          setIsInputDisabled(true)
+          setVenueHasPricingPoint(true)
+          setIsBannerVisible(false)
+          setIsConfirmSiretDialogOpen(false)
+        })
+    }
+  }
+  const pricingPointOptions = [
+    { value: '', label: 'Sélectionner un lieu dans la liste' },
+    ...offerer.managedVenues
+      .filter(venue => venue.siret)
+      .map(venue => ({
+        value: venue.nonHumanizedId,
+        label: `${venue.publicName || venue.name} - ${venue?.siret}`,
+      })),
+  ]
+
+  return (
+    <div className="section vp-content-section bank-information">
+      <div className="main-list-title">
+        <h3 className={`${styles['title']}`}>Barème de remboursement</h3>
+      </div>
+      {!readOnly && !venue.pricingPoint && isBannerVisible && (
+        <Banner
+          links={[
+            {
+              href: `https://aide.passculture.app/hc/fr/articles/4413973462929--Acteurs-Culturels-Comment-rattacher-mes-points-de-remboursement-et-mes-coordonn%C3%A9es-bancaires-%C3%A0-un-SIRET-de-r%C3%A9f%C3%A9rence-`,
+              linkTitle: 'En savoir plus sur les barèmes de remboursement',
+              Icon: ExternalSiteIcon,
+            },
+          ]}
+          type="notification-info"
+        >
+          Si vous souhaitez vous faire rembourser les offres de votre lieu sans
+          SIRET, vous devez sélectionner un lieu avec SIRET dans votre structure
+          afin de permettre le calcul de votre barème de remboursement.
+          Attention, vous ne pourrez plus modifier votre sélection après
+          validation.
+        </Banner>
+      )}
+
+      {isConfirmSiretDialogOpen && (
+        <ConfirmDialog
+          cancelText={'Annuler'}
+          confirmText={'Valider ma sélection'}
```

Comment:
```suggestion
          cancelText='Annuler'
          confirmText='Valider ma sélection'
```
Pas besoin des `{}` pour passer du texte en props :)